src/core/remote.js

```diff
@@ -1822,6 +1835,10 @@ Remote.prototype.findAccount = function(accountID) {
  */
 Remote.prototype.createPathFind = function(options, callback) {
   if (this._cur_path_find !== null) {
+    if (callback === undefined) {
```

Comment:
Yeah, maybe that approach would get too complicated. I think we should change the error message though. "Only one streaming pathfind request at a time is supported"